app/controllers/histories_controller.rb

```diff
@@ -46,16 +46,24 @@ def check_role
   end
 
   def check_patient
-    @patient = current_user.viewable_patients.find_by(id: params.require(:patient_id))
-    return head :forbidden if @patient.nil?
+    patient_id = params.require(:patient_id)&.to_i
+
+    # Check if Patient ID is valid
+    unless Patient.exists?(patient_id)
+      error_message = "History comment cannot be modified for unknown monitoree with ID: #{patient_id}"
+      render(json: { error: error_message }, status: :bad_request) && return
```

Comment:
Is the `error_message` variable just for readability? My preference would be to just using the string literal directly in the JSON. Ditto for the handful of other instances in the two controllers here where this is done.

Reply:
ultimately yes because the line was too long and had to be broken up anyway.... this was the cleaner version IMO

Reply:
I'm generally not a fan of creating variables to be used once, unless readability is really bad, and I think the readability is fine with the string literal. It's not really worth debating though, so I'm gonna just close this thread.

Reply:
I did end up cleaning this part up: https://github.com/SaraAlert/SaraAlert/pull/900/commits/a3cd41ef431c8db20bb05e0b5d49fac393d0e383